### DISCUSSION TITLE
How to detect when freeswitch is down using Homer?

### DISCUSSION THREAD

**Discussion by ghost** [sipcapture/homer#494] [Q&A] [answered]
Hello World!!!

I'm newbie to homer and currently I'm using freeswitch wrapped around fusionpbx where the deployed homer(heplify) agent is sending packets to homer 7 server. I want to detect when the fusionpbx(freeswitch) server would go down using homer so that I could instantly switch to another fusionpbx server. How could I detect such behaviour using homer?

many Thanks

**Comment by adubovikov** [comment, marked as answer]
1. You can use prometheus to see if you don't have any HEP packets in stats.
2. You can use monit for this.
